core/src/main/java/bisq/core/trade/protocol/BuyerAsMakerProtocol.java

```diff
@@ -100,8 +100,8 @@ protected void handle(DelayedPayoutTxSignatureRequest message, NodeAddress peer)
                 .with(message)
                 .from(peer))
                 .setup(tasks(
-                        BuyerProcessDelayedPayoutTxSignatureRequest.class,
                         MakerRemovesOpenOffer.class,
+                        BuyerProcessDelayedPayoutTxSignatureRequest.class,
```

Comment:
Could this be abused by sending a bad request, causing offers to be removed but guaranteed failed trade?

Reply:
Was thinking the same, but a taker could do it anyway at any point later as well. We could add some feature to restore offers which got closed from a trade failure by cloning  the offer and change the id. So that would render such an attack as ineffective.

Reply:
Yeah, it's an area that needs some more thought and this is not the real issue.